### PR TITLE
Update of tutorial sections

### PR DESCRIPTION
**Pull request by edgarmueller**
@koegel Please have a look at the udpated wording, if you find the time.

### CODE REVIEW

File: src/pages/docs/listings/seed.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-git clone https://github.com/eclipsesource/jsonforms-react-seed
-cd jsonforms-react-seed
+git clone https://github.com/eclipsesource/jsonforms-react-seed jsonforms-custom-renderers-tutorial
```
Comment (eneufeld) [inline src/pages/docs/listings/seed.md:1]: is the clone command correct?

File: src/pages/docs/Rating.jsx [hunk 2]
```diff
@@ -10,6 +10,16 @@ export class Rating extends React.Component {
     };
   }
 
+  static getDerivedStateFromProps = (nextProps, prevState) => {
```
Comment (eneufeld) [inline src/pages/docs/Rating.jsx:13]: is this a method of the React.Component?

**Review by eneufeld — APPROVED** [review]
looks good, just have 2 small questions

File: src/pages/docs/listings/seed.md (hunk 1 above)
Comment (edgarmueller) [reply src/pages/docs/listings/seed.md:1]: As discussed.

File: src/pages/docs/Rating.jsx (hunk 2 above)
Comment (edgarmueller) [reply src/pages/docs/Rating.jsx:13]: Yes, see https://reactjs.org/docs/react-component.html#static-getderivedstatefromprops

**Pull request merged**